dumpling/export/dump.go

```diff
@@ -443,7 +452,10 @@ func adjustDatabaseCollation(tctx *tcontext.Context, parser *parser.Parser, orig
 }
 
 // adjustTableCollation adjusts table collation
-func adjustTableCollation(tctx *tcontext.Context, parser *parser.Parser, originSQL string, charsetAndDefaultCollationMap map[string]string) (string, error) {
+func adjustTableCollation(tctx *tcontext.Context, collationCompatible string, parser *parser.Parser, originSQL string, charsetAndDefaultCollationMap map[string]string) (string, error) {
+	if collationCompatible != StrictCollationCompatible {
```

Comment:
maybe do this check out of adjustTableCollation?  not strong , current code lgtm too

Reply:
yes, i do this like that before, but i find it is maybe hard to add UT for this config, because the judgement will be in `dumpDatabases`.

Reply:
ok, maybe we can enhance this later, because codefreeze is coming, i will merge this pr first